### PR TITLE
update README with instructions

### PR DESCRIPTION
**Pull request by marycrawford**
# DEVOPS PULL REQUEST

## Related Issue

- resolve CDCgov/dibbs-cloud/issues/100

## Changes Proposed

- Updates README.md with table of contents
- Updates README.md with documentation for partners to utilize dibbs-aws repository in their local AWS environments

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,33 +56,173 @@ WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
...
+## 4.5 Update Terraform Through The Command Line
+
+This section will go over some of the sections you will need to amend or change in your local terraform branch.
+
+<em><strong>4.5.1. Update And Setup Your AWS Backend</em></strong>
+* In your terminal, navigate to the _/terraform/setup_ folder (`cd /terraform/setup`).  
+* Run `./setup.sh`.
+* Now set your variable values for (1) *Region*, (2) *Owner*, and (3) *Project* when prompted in the command line.  
+&nbsp;&nbsp;&nbsp;&nbsp;<em><strong>Note</em></strong>:  The default value for Owner is `Skylight`.  You can change this value to an \
+&nbsp;&nbsp;&nbsp;&nbsp;abbreviated version of your organization (i.e. `WA-STILTS`).  _The Owner name \
```
Comment (rin-skylight) [inline README.md:170]: I think the abbreviation you're looking for here is "STLT". This wouldn't necessarily make sense in this context, though...perhaps consider something like "You can change this value to one that represents your organization or department."

File: README.md [hunk 2]
```diff
@@ -26,33 +56,173 @@ WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
...
+
++ [Return to Table of Contents](#table-of-contents).
+
+## 2.6 Additional Standard Notices
 Please refer to [CDC's Template Repository](https://github.com/CDCgov/template) for more information about [contributing to this repository](https://github.com/CDCgov/template/blob/main/CONTRIBUTING.md), [public domain notices and disclaimers](https://github.com/CDCgov/template/blob/main/DISCLAIMER.md), and [code of conduct](https://github.com/CDCgov/template/blob/main/code-of-conduct.md).
+
+
++ [Return to Table of Contents](#table-of-contents).
+
+# 3. Architectural Design
```
Comment (rin-skylight) [inline README.md:97]: This section is a bit confusing...why are we including both current and final? I think we should focus on the diagram that directly reflects what is included in this repository at present. That way, there's no confusion.

**Review by rin-skylight — CHANGES_REQUESTED** [review]
This looks great overall! Thank you for making it so well-organized and clear. I've only found two small things that need to be addressed, but everything else is good to go!

File: README.md [hunk 3]
```diff
@@ -26,33 +56,173 @@ WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
...
+
+[Return to Table of Contents](#table-of-contents).
+
+## 4.5 Update Terraform Through The Command Line
+
+This section will go over some of the sections you will need to amend or change in your local terraform branch.
+
+<em><strong>4.5.1. Update And Setup Your AWS Backend</em></strong>
+* In your terminal, navigate to the _/terraform/setup_ folder (`cd /terraform/setup`).  
+* Run `./setup.sh`.
```
Comment (alismx) [inline README.md:167]: I think some of this `/setup` will need to be updated. The current version of this script will attempt to do several things.

- Create a tfvars file
- Create the AWS resources to store the terraform state + Create some local files to help with setting up the ECS infrastructure.
- Update the terraform backend to push the state that was just created

The only time they should have to manually init would be if:

- They want to
- The script fails
- They are upgrading terraform providers

I'd be happy to demo this for the team and to help clear up the docs here.

**Review by alismx — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (marycrawford) [reply README.md:97]: Good point!  I updated the diagram and removed the future architectural design.

File: README.md (hunk 1 above)
Comment (marycrawford) [reply README.md:165]: Great suggestions!  I have updated the documentation to make this clearer for our partners.

File: README.md (hunk 3 above)
Comment (marycrawford) [reply README.md:162]: Per our discussion, I made the revisions and will await your review prior to resolving this comment.

**Review by alismx — APPROVED** [review]
Looks good, thanks for working on this! :shipit:

**Review by rin-skylight — APPROVED** [review]

**Pull request merged**